### DISCUSSION TITLE
Entità perse su Alexa

### DISCUSSION THREAD

**Discussion by NTorto** [indomus/forum#1362] [HUB: Home Assistant]
Ciao a tutti,
ho integrato con successo Alexa ad home assistant tramite AWS, seguendo l'ottima guida di InDomus, da questa estate e tutto funzionava perfettamente.

Questa mattina ho deciso di aggiornare HA (che era fermo alla 2021.7) alla versione 2021.12.10. 
Dopo l'aggiornamento però tutte le entità esposte verso alexa sono irrangiungibili da questa, e chiaramente continuano a funzionare correttamente in HA.
Continuo a poter controllare l'echo dot da HA e sui log non ci sono messaggi di errore a riguardo.
Cosa potrei provare?

Vi ringrazio per l'aiuto.

**Comment by snakuzzo** [comment]
Allora...innanzi tutto il controllo degli echo non c'entra niente con l'integrazione con haaska su AWS.
Gli echo li piloti con un custom component e non con quella integrazione.
Per quanto riguarda l'indisponibilità dei device lato Alexa, ti conviene:

- andare sulla console AWS e testare la lambda function
- verificare i log di Core

Non mi pare ci siano state breaking in merito da ottobre ad oggi

**Reply by NTorto** [reply]
Grazie per il consiglio di AWS snakuzzo: ho rifatto la prova da lì e il test ha fallito; guardando il log ho visto che l'hub non era più raggiungibile dall'esterno della rete. Infatti provando da rete esterna non lo raggiungevo (stamattina stupidamente non ho fatto questa prova :P).
Ho dovuto riconfigurare i container di docker compose sia di duckdns che di caddy, i quali non so perchè erano caduti.
Ora va.
Grazie mille.
